bitnami/airflow/templates/web/deployment.yaml

```
@@ -240,6 +246,9 @@ spec:
             {{- if .Values.web.extraVolumeMounts }}
             {{- include "common.tplvalues.render" (dict "value" .Values.web.extraVolumeMounts "context" $) | nindent 12 }}
             {{- end }}
+        {{- if and .Values.dags.enabled (not .Values.dagProcessor.enabled) }}
+        {{- include "airflow.defaultSidecars.syncDAGs" . | nindent 8 }}
+        {{- end }}
```

Simplify this condition:

```suggestion
        {{- if and .Values.dags.enabled }}
        {{- include "airflow.defaultSidecars.syncDAGs" . | nindent 8 }}
        {{- end }}
```

We should only exclude it from the Scheduler if Dag Processor is enabled.